config.json

```diff
@@ -19,6 +19,13 @@
       ]
     },
     {
+      "slug": "rotational-cipher",
+      "difficulty": 1,
```

Comment:
Since you implemented this exercise you would probably be in a good position to estimate the difficulty if you want to :) Just change the difficulty number and move this entry that it's with the other exercises with the same difficulty. 

But don't worry if you don't want to estimate it, you can just leave it with difficulty 1 and we'll estimate it later :) It would be great if you could move it to the end of this list in that case though. That way it'll be positioned at the end of the track so at least users don't get it too early :) With it's current position this would be the third exercise users get which is probably too early.